try:
	print(sys.path)
except Exception, e:
	print('In exception')
	print e;
